### PR TITLE
Week2 synthesizing

### PR DESCRIPTION
**Pull request by Lalanikw**
Hi Professor, 
Here is my task1/2. Task3 is not pushing as I expected. I will try again. 
Lalani.

### CODE REVIEW

**Comment by Lalanikw** [comment]
Task03 also added. 
Thanks, 
Lalani.

File: task_01.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#################
+Why I Love Python
+#################
+
+Why I love **Python**, a language worth knowing.
+
+********************************
+Running It
+********************************
+Python syntax is really quite simple to execute. Anyone can do it just by executing the python command on the command line and invoking the interpreter.
```
Comment (cheuschober) [inline task_01.rst]: In general, always put a full blank line between a heading and anything following. You also want your headers to have the same length over and underlines as their text.

File: task_01.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Why I love **Python**, a language worth knowing.
+
+********************************
+Running It
+********************************
+Python syntax is really quite simple to execute. Anyone can do it just by executing the python command on the command line and invoking the interpreter. 
+
+You can also execute python programs by calling the program filename such as: 
+
+me@mymachine $ python my_file_name.py
```
Comment (cheuschober) [inline task_01.rst]: Try looking at the `.. code:: console` and `.. code:: pycon` directives for helping highlight these elements.

File: task_01.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+**********************
+The People
+**********************
+
+It's amazing how nice people are in the Python developement worls. Just as with any community, however, it's important to follow certain rules of decorum. 
+
+* Be courteous to others
+* Always read the manual before asking questions
+* Be succinct in your questions
+	+Don't give unnecessary details if they aren't needed
```
Comment (cheuschober) [inline task_01.rst]: Spacing is VERY important in both restructuredtext and python. The difference here between `+Don't` and `+ Don't` is huge. It's a particular form of detail to which you must pay close attention.

File: task_01.rst [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+|**Name**  |**Contact**    |
++==========+===============+
+|Me        |me@mysite.com  |
++----------+---------------+
+|Myself    |myself@me.com  |
++----------+---------------+
+|I         |i@mii.com      |
++----------+---------------+
+
++------------------------------------------------------------+
```
Comment (cheuschober) [inline task_01.rst]: Look into the `.. note::` directive for this in the future.

**Comment by cheuschober** [comment]
In the case of task_03 it looks like you ignored your unit test result! Make sure you pay attention to both lint and unit test results now and in the future!

**Pull request merged**